### PR TITLE
Create layouts page

### PR DESCRIPTION
**Pull request by jmestxr**
Create layouts page consisting of the following layouts:
- Template grid
- Side navigation area
- Content area
- Table-of-content area

### CODE REVIEW

File: portal/_data/navigation.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@
 - title: foundations
   url: /foundations/
   has-collection: true
+- title: layouts
```
Comment (rav3) [inline portal/_data/navigation.yml]: @jmestxr 
Could you shift the `Layout` menu in between `Patterns` and `Templates`?
`Components` `Patterns` `Layouts` `Templates` in this order

**Comment by jmestxr** [comment]
@rav3 The changes you requested are made.

**Comment by clukhei** [comment]
@rav3 do we need to use png for images or svg also can? 

i downloaded the png to replace, but the resolution is low. 
Group 1215

svg on the other hand clearer, but the dotted lines is crooked
Screenshot 2023-12-28 at 4 36 13 PM

**Comment by clukhei** [comment]
Added image compressor plugin

File: portal/assets/img/layout/template-grid/example-only-content.png
Comment (rav3) [inline portal/assets/img/layout/template-grid/example-only-content.png:1]: Theres an excess transparency at the bottom.

**Review by rav3 — APPROVED** [review]

**Review by darenng — APPROVED** [review]

**Pull request merged**